scripts/ci/update-changelog.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -ex
+
+
+dependencies="cargo git git-cliff"
+
+for dependency in $dependencies
+do
+  if ! command -v $dependency &>/dev/null
+  then
+    echo "error: $dependency was not found in PATH" >&2
+    exit 255
+  fi
+done
+
+# The default "starting" commit is a somewhat arbitrary starting point for
+# cataloging recent commits in a way that breaks from the old convention
+DEFAULT_START_COMMIT="891e7d47827e99947e46c82f509e479aa13acf24"
```

Comment:
Should this be `e91d41810f3f44061db544e771f0fc14515aff84`? Or something else on the `2.0` branch?

Reply:
oh hmm, I thought I changed it to the 2.0.8 commit. i'll fix